### PR TITLE
Added conversion for WordpieceTokenizer

### PR DESCRIPTION
**Pull request by xadupre**
Signed-off-by: xavier dupré <xavier.dupre@gmail.com>

### CODE REVIEW

File: tf2onnx/graph.py [hunk 1]
```diff
@@ -1343,6 +1343,12 @@ def find_output_consumers(self, output_name):
                 nodes.extend(g.find_output_consumers(output_name))
         return nodes
 
+    def remove_output(self, output_name):
```
Comment (TomWildenhain-Microsoft) [inline tf2onnx/graph.py:1346]: Looks like this is never used.

File: tf2onnx/custom_opsets/string_ops.py [hunk 2]
```diff
@@ -168,3 +170,41 @@ def version_1(cls, ctx, node, **kwargs):
...
+        table = initialized_tables[ressource]
+        if isinstance(table, tuple):
+            table = table[0]
+        mapping = {}
+        for i, word in enumerate(table):
+            if isinstance(word, bytes):
+                word = word.decode('utf-8')
+            mapping[word] = i
+        st = io.StringIO()
+        json.dump(mapping, st, separators=(',', ':'))
```
Comment (TomWildenhain-Microsoft) [inline tf2onnx/custom_opsets/string_ops.py:206]: I think you can use `json.dumps` instead if you want a string.

**Review by TomWildenhain-Microsoft — CHANGES_REQUESTED** [review]
A few comments, otherwise looks good!

File: tf2onnx/graph.py (hunk 1 above)
Comment (xadupre) [reply tf2onnx/graph.py]: removed

**Review by TomWildenhain-Microsoft — APPROVED** [review]

**Comment by guschmue** [comment]
@xadupre , what this merged ?

**Pull request merged**